packages/lib-content/src/screens/Home/DefaultHome/components/FeaturedProjects.js

```diff
@@ -0,0 +1,91 @@
+/* Note that this component will also be used in the redesigned Projects landing page
+so it could be moved to lib-react-components at that time */
+
+import { Box, Heading, ResponsiveContext } from 'grommet'
+import { useContext } from 'react'
+import useSWR from 'swr'
+import { useTranslation } from '../../../../translations/i18n.js'
+import { projects } from '@zooniverse/panoptes-js'
+import { ProjectCard, SpacedHeading } from '@zooniverse/react-components'
+
+import HorizontalRuleLogo from '../../../../components/HorizontalRuleLogo/HorizontalRuleLogo.js'
+
+const SWROptions = {
+  revalidateIfStale: true,
+  revalidateOnMount: true,
+  revalidateOnFocus: true,
+  revalidateOnReconnect: true,
+  refreshInterval: 0
+}
+
+const getFeaturedProjects = async () => {
+  try {
+    const response = await projects.get({
+      query: {
+        featured: true,
+        launch_approved: true,
+        cards: true
+      }
+    })
+    if (response.ok) {
+      return response.body.projects
+    }
+    return []
+  } catch (error) {
+    console.log(error)
+    return []
+  }
+}
+
+function useFeaturedProjects() {
+  return useSWR('featured-projects', getFeaturedProjects, SWROptions)
+}
+
+export default function FeaturedProjects() {
+  const { data: featuredProjects, isLoading } = useFeaturedProjects()
+  const { t } = useTranslation()
+
+  // A breakpoint x-large needs to be implemented at the theme level
+  // Then on x-large can be passed to ProjectCard below
+  const size = useContext(ResponsiveContext)
```

Comment:
I want to be able to simply past `size` to ProjectCard, which is a prop that determines its width. We don't have an `x-large` breakpoint in FEM's Grommet theme yet, so that will need to be added in a separate PR. For now, the featured projects section has cards sized large, medium, and small.

Reply:
I actually decided to just write the ProjectCard size logic directly in FeaturedProjects rather than rely on an 'xlarge' Grommet theme breakpoint. See https://github.com/zooniverse/front-end-monorepo/pull/6109 for our current breakpoint sizes.